.gitignore

```diff
@@ -0,0 +1,9 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Perhaps remove this? I generally don't use IntelliJ when making HTML pages, and it isn't a compiled language so there shouldn't be compile directory.